src/app/models/matchups.model.ts

```diff
@@ -0,0 +1,62 @@
+export interface WeeklyMatchupInfo {
+  week: number,
+      squadMatchups: Array<{weeklyMatchup: Array<Matchups>}>,
+      active: boolean,
+      past: boolean,
+}
+
+export interface Matchups {
+  squadUID: string,
+  selectedSquads: Array<
+  {
+    category: string,
+    team: string,
+    points: number
+  }
+  >
+  squadName?: string;
+  matchupTotal: number
+}
+
+export interface MatchUpMetrics {
+  passingTeam: {
+    netPassingYards: MatchupScoring,
+    passingTDs: MatchupScoring
+  },
+  rushingTeam: {
+    rushingYards: MatchupScoring,
+    rushingTDs: MatchupScoring
+  },
+  defensiveTeam: {
+    sacks: MatchupScoring,
+    interceptions: MatchupScoring,
+    fumblesRecovered: MatchupScoring,
+    defensiveTDs: MatchupScoring
+  }
+}
+
+export interface MatchupScoring {
+      name: string,
+      value: number,
+      perYard: number,
+      total: boolean
```

Comment:
total is supposed to be a boolean?

Reply:
It might not be the best word but it tells us if I am using the total value or a per yard to calculate the score. Check out where I use it and the difference it makes in the calculation.